src/test/mir-opt/const_prop/usize_literal_propagation/32bit/rustc.main.ConstProp.diff

```diff
@@ -0,0 +1,55 @@
+- // MIR for `main` before ConstProp
```

Comment:
I think this file can be removed right since the original test was renamed?

Reply:
Omg, git got me again. Patches do not commute 🤦🏻

I had deleted this in the last changes and then decided to rebase them up the tree so that we had the testing changes all sort of together. But then git decided that this change was reverted. 🤦🏻

Thanks for the sanity check!